pkg/helm/client.go

```diff
@@ -128,8 +128,18 @@ func (h CliClient) UpdateRepo(ctx context.Context) error {
 	return nil
 }
 
-// Install installs helm chart with given release name
-func (h CliClient) Install(ctx context.Context, chart, version, release, namespace string, values map[string]string, wait bool) error {
+// Install installs a Helm chart in the specified namespace with the given release name and chart version.
+// `wait` and `dryRun` can be set to `true` to make sure it adds `--wait` and `--dry-run` flags to the
+// `helm install command`.
```

Comment:
```suggestion
// `helm install` command.
```

Reply:
@mabhi can you please address this and then we can merge.

Reply:
Done